File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q03.py
import math
def main():
    #Input
    minutes = int(input("Enter a value in minutes: "))

    #Processing
    hours = (minutes / 60)
    value_hours = math.floor(hours)
    value_minutes = minutes % 60

    #Output
    print(f"The entered value of {minutes} minutes converted in hours and minutes is equal to: {value_hours} hour(s) and {value_minutes} minute(s)")
    

main()

File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q03.js
import { question } from "readline-sync";
function main(){
    //Input
    const minutes = parseInt(question("Enter a value in minutes: "))

    //Processing
    const value_hours = Math.floor(minutes / 60)
    const value_minutes = minutes % 60
    //Output
    console.log(`The minutes entered conveterd in hours and minutes are: ${value_hours}`)
}
main()